File: community/training/debugging/first_demo.py
def first_function():
    total = 0
    for x in range(1, 50):
        y = second_function(x)
        total = total + y

    return total


def second_function(a):
    result = 3 * a**2 + 5 * a
    return result


total = first_function()
print(f'Total = {total}')

File: pr-preview/pr-78/community/training/debugging/first_demo.R
first_function <- function() {
  total <- 0
  for (x in seq(49)) {
    y <- second_function(x)
    total <- total + y
  }
  total
}

second_function <- function(a) {
  result <- 3 * a^2 + 5 * a
  result
}

total <- first_function()
cat("Total =", total, "\n")
